frontend/app/composables/useLogin.ts

```diff
@@ -0,0 +1,38 @@
+import { error } from "#build/ui";
+
+export interface User {
+    uuid: string;
+    userId: string;
+    name: string;
+    mail: string;
+    profile: string;
+    birthDate: string;
+    iconImage: string;
+    password: string;
+}
+
+export interface LoginResponse {
+    succes: boolean,
+    message: string,
+    User: User,
+}
+
+export const useLogin = () => {
+    const login = async (email: string, password: string): Promise<LoginResponse> => {
+        try {
+            const res: LoginResponse = await $fetch('http://localhost:8080/user/login', {
+                method: 'POST',
+                body: {
+                    mail: email,
+                    password: password,
+                },
+            })
+            return res;
+        } catch (erorr) {
+            console.log(error);
+            throw error;
+        }
+    }
+
+    return { login };
+}
```

Comment:
ファイル末尾には空行を入れるようにしてください。
```suggestion
}

```